lib/msal-node/test/client/PublicClientApplication.spec.ts

```diff
@@ -29,6 +29,17 @@ import {
     AuthorityFactory,
```

Comment:
I believe we re-use the same silent flow tests in `ConfidentialClientApplication.spec.ts`. Do you need to update those too?

Reply:
Nope, looks like the silent flow test in ConfidentialClientApplication.spec.ts doesn't need to change in response to this PR